### PR TITLE
OO-744 adding a label to the korb in lieferplanung when zusatzabo

### PR DESCRIPTION
**Pull request by mcmpp**

### CODE REVIEW

File: app/scripts/lieferplanungen/detail/korbplanung.html [hunk 1]
```diff
@@ -85,6 +85,7 @@ <h3 class="panel-title lieferplanung-title">
               <span class="btn btn-default btn-sm"
                 oo-draggable="true" oo-draged-type="'korb'"><span class="fa fa-files-o"></span></span>
               <oo-delete-button model="abotypLieferung" entity="Lieferung" ng-show="valuesEditable()" on-delete="removeAbotypFromPlanung" button-class="btn-default" confirm="true" condensed="true"></oo-delete-button>
+              <div ng-if="!abotypLieferung.abotyp.anzahlAbwesenheiten" > Zusatzabo</div>
```
Comment (AlwinEgger) [inline app/scripts/lieferplanungen/detail/korbplanung.html]: I think you got the wrong field. Why check on "anzahlAbwesenheiten" on the abotyp?

Comment (AlwinEgger) [reply app/scripts/lieferplanungen/detail/korbplanung.html]: Ok after discussion.

**Review by AlwinEgger — APPROVED** [review]

**Comment by mcmpp** [comment]
reopen this pull request fixing the following links for the zusatzabos: 

- Anzahl Körbe zu liefern
- Anzahl Abwesenheiten
- Anzahl Saldo zutief

File: app/scripts/lieferplanungen/lieferplanung.model.js [hunk 2]
```diff
@@ -75,6 +75,11 @@ angular.module('openolitor-admin')
         isArray: true,
         url: API_URL + 'lieferplanungen/:id/lieferungen/:lieferungId/:korbStatus/aboIds'
       },
+      'getZusatzaboIdsByKorbStatus': {
+        method: 'GET',
+        isArray: true,
+        url: API_URL + 'lieferplanungen/:id/lieferungen/:lieferungId/:korbStatus/zusatzaboIds'
```
Comment (AlwinEgger) [inline app/scripts/lieferplanungen/lieferplanung.model.js]: What you actually get are "HauptaboIds", no? So the route could be "lieferplanungen/:id/lieferungen/:lieferungId/:korbStatus/hauptaboIds"

Comment (mcmpp) [reply app/scripts/lieferplanungen/lieferplanung.model.js]: you are right! I'll change it

**Review by AlwinEgger — APPROVED** [review]

**Pull request merged**